### PR TITLE
Copy core artifacts so periphery can deploy core contracts without exposing core code.

### PR DESCRIPTION
**Pull request by TBomberman**
Update hash
Added tokens script to send to your metamask.

### CODE REVIEW

**Comment by TBomberman** [comment]
image
Deploys ok.

File: scripts/supplyTokens.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    to: metamaskAddress,
+    value: ethers.utils.parseEther("100.0"), // Sends exactly 100.0 ether
+  });
+  
+  const tokenA = await ethers.getContractAt("TestERC20", tokenAaddr);
+  const tokenB = await ethers.getContractAt("TestERC20", tokenBaddr);
+  await tokenA.transfer(metamaskAddress, ethers.utils.parseEther("100.0"));
+  await tokenB.transfer(metamaskAddress, ethers.utils.parseEther("100.0"));
+}
+
```
Comment (Pochetes) [inline scripts/supplyTokens.ts:19]: why is this file needed? can't we generate fake addresses with ether using `npx hardhat node` and then import those accounts in metamask to use?

**Review by Pochetes — COMMENTED** [review]
minor comments

File: scripts/supplyTokens.ts (hunk 1 above)
Comment (TBomberman) [reply scripts/supplyTokens.ts]: You are right, removed this file, and updated readme.

**Review by Pochetes — APPROVED** [review]
LGTM. Let's continue the `artifacts-core` discussion when we're closer to releasing to testnet.
